matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/summary/RoomSummaryConstants.kt

```diff
@@ -33,5 +33,5 @@ object RoomSummaryConstants {
             EventType.ENCRYPTED,
             EventType.STICKER,
             EventType.REACTION
-    ) + EventType.POLL_START
+    ) + EventType.POLL_START + EventType.STATE_ROOM_BEACON_INFO
```

Comment:
looks like these event types could be inside the `listOf()`

Reply:
In fact these event types are not of type `String` values but of type `List<String>` in order to support stable/unstable fields. That's why they are added to the `listOf().

Reply:
ahhhhhhh thanks for explaining!